app/src/main/java/com/hieuminh/chatapp/dialog/LoadingDialog.kt

```diff
@@ -0,0 +1,26 @@
+package com.hieuminh.chatapp.dialog
+
+import android.annotation.SuppressLint
+import android.app.Activity
+import android.app.AlertDialog
+import com.hieuminh.chatapp.R
+
+class LoadingDialog(private val activity: Activity) {
+
+    private lateinit var alertDialog: AlertDialog
```

Comment:
```suggestion
    private lateinit var mAlertDialog: AlertDialog
```
put 'm' before global variable to distinguish local and global variable